jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -51,6 +75,21 @@ def parse_data(app, file_obj, ext=None, data_label=None):
                 pf = rgb2gray(im)
             pf = pf[::-1, :]  # Flip it
             _parse_image(app, pf, data_label, ext=ext)
+
+        elif file_obj_lower.endswith('.asdf'):
+            # First check if file might be a Roman data product.
+            with asdf.open(file_obj) as asdf_file:
+                # This is a convention of roman data products.
+                if 'roman' in asdf_file:
```

Comment:
any chance this logic could break (roman appearing in `asdf_file` for another reason like observer's last name)?